apps/web/components/Embed.tsx

```diff
@@ -378,14 +378,14 @@ const embeds: {
         <rect x="85" y="161.5" width="6" height="6" rx="1" fill="#C6C6C6" />
         <rect x="97" y="161.5" width="6" height="6" rx="1" fill="#3E3E3E" />
         <rect x="109" y="161.5" width="6" height="6" rx="1" fill="#C6C6C6" />
-        <rect x="24.5" y="51" width="139" height="163" rx="1.5" stroke="#292929" />
-        <rect x="176" y="50.5" width="108" height="164" rx="2" fill="#E1E1E1" />
-        <rect x="24" y="226.5" width="260" height="38.5" rx="2" fill="#E1E1E1" />
+        <rect x="24.5" y="51" width="139" height="163" rx="6" stroke="#292929" />
+        <rect x="176" y="50.5" width="108" height="164" rx="6" fill="#F3F4F6" />
+        <rect x="24" y="226.5" width="260" height="38.5" rx="6" fill="#F3F4F6" />
       </svg>
     ),
   },
   {
-    title: "Floating pop-up button",
+    title: "Floating pop-up Button",
```

Comment:
In the Figma design it's spelled like that:
<img width="281" alt="Screenshot 2023-02-24 at 11 47 37" src="https://user-images.githubusercontent.com/30310907/221237836-b8f35908-2a54-4db9-baf9-e886843b0da0.png">
@Jaibles Should it really be the uppercase U in pop Up?

Reply:
Hi @Jaibles, what should it be?

Reply:
Floating pop-up button